x-pack/plugins/enterprise_search/public/applications/app_search/components/role_mappings/role_mappings.tsx

```diff
@@ -22,15 +22,15 @@ import {
 } from '../../../shared/role_mapping';
 import { ROLE_MAPPINGS_TITLE } from '../../../shared/role_mapping/constants';
 
-import { DOCS_PREFIX } from '../../routes';
+import { SECURITY_DOCS_URL } from '../../routes';
 import { AppSearchPageTemplate } from '../layout';
 
 import { ROLE_MAPPINGS_ENGINE_ACCESS_HEADING } from './constants';
 import { RoleMapping } from './role_mapping';
 import { RoleMappingsLogic } from './role_mappings_logic';
 import { User } from './user';
 
-const ROLES_DOCS_LINK = `${DOCS_PREFIX}/security-and-users.html`;
+const ROLES_DOCS_LINK = SECURITY_DOCS_URL;
```

Comment:
You could just remove `ROLES_DOCS_LINK` and use `SECURITY_DOCS_URL` directly instead